src/content/docs/fr/guides/deploy/render.mdx

```diff
@@ -5,14 +5,15 @@ type: deploy
 i18nReady: true
 ---
 
-Vous pouvez déployer votre projet Astro sur [Render](https://render.com/), un service pour build des sites web avec des certificats libres TLS, un CDN global, une protection DDoS, des réseaux privés et des déploiements automatiques depuis Git.
+Vous pouvez déployer votre projet Astro sur [Render](https://render.com/), un service de création de sites web avec des certificats TLS gratuits, un CDN global, une protection DDoS, des réseaux privés et des déploiements automatiques à partir de Git.
 
 ## Comment déployer
 
-1. Créer un [compte render.com](https://dashboard.render.com/) et se connecter
-2. Cliquer sur le bouton **Nouveau +** du tableau de bord et sélectionner **Site Statique**
-3. Connecter votre dépôt [GitHub](https://github.com/) ou [GitLab](https://about.gitlab.com/) ou, alternativement saisir l'adresse publique d'un dépôt publique
-4. Donner à votre site web un nom, sélectionner la branche et spécifier la commande de génération et le dossier de publication
-   - **commande de génération:** `npm run build`
-   - **dossier de publication:** `dist`
-5. Cliquer sur le bouton **Créer un Site Statique**
+1. Créer un [compte render.com](https://dashboard.render.com/) et connectez-vous
+2. Cliquer sur le bouton **New +** du tableau de bord et sélectionner **Static Site**
+3. Connectez votre dépôt [GitHub](https://github.com/) ou [GitLab](https://about.gitlab.com/) ou entrez l'URL publique d'un dépôt public.
+4. Donnez un nom à votre site web, sélectionnez la branche et spécifiez la commande de construction et le répertoire de publication
+   - **build command:** `npm run build` (Commande de construction)
+   - **publish directory:** `dist` (Répertoire de publication)
+   - **Environment variables (advanced)**: (Variables d'environnement) Par défaut, Render utilise Node.js 14.17.0, mais Astro [nécessite une version supérieure](/fr/install/auto/#prérequis).  Ajoutez une variable d'environnement avec une **Variable key** (clé de variable) de `NODE_VERSION` et une **Value** (Valeur) de `18.14.1` ou plus pour dire à Render d'utiliser une version compatible de Node.js. Alternativement, ajoutez un fichier [`.node-version`](https://render.com/docs/node-version) ou [`.nvmrc`](https://render.com/docs/node-version) file to your project to specify a Node.js version.
```

Comment:
`file to your project to specify a Node.js version.`
Hello ! Il manque de la traduction par ici!